src/main/java/com/google/devtools/build/lib/syntax/FuncallExpression.java

```diff
@@ -731,26 +731,22 @@ private boolean includeSelfAsArg(
   /**
    * Call a method depending on the type of an object it is called on.
    *
-   * @param positionals The first object is expected to be the object the method is called on.
+   * @param positionalArgs The first object is expected to be the object the method is called on.
```

Comment:
This is no longer necessarily true. Could you please update the doc?

Reply:
good catch!